frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -62,18 +62,18 @@ export default class ProfileLink extends Component {
         });
 
         return (
-            <OnClickOutsideWrapper handleDismissal={this.closeDropdown}>
-                <div className={dropDownClasses}>
-                    <a data-metabase-event={"Navbar;Profile Dropdown;Toggle"} className="NavDropdown-button NavItem flex align-center p2 transition-background" onClick={this.toggleDropdown}>
-                        <div className="NavDropdown-button-layer">
-                            <div className="flex align-center">
-                                <UserAvatar user={user} style={{backgroundColor: 'transparent'}}/>
-                                <Icon name="chevrondown" className="Dropdown-chevron ml1" size={8} />
-                            </div>
+            <div className={dropDownClasses}>
+                <a data-metabase-event={"Navbar;Profile Dropdown;Toggle"} className="NavDropdown-button NavItem flex align-center p2 transition-background" onClick={this.toggleDropdown}>
+                    <div className="NavDropdown-button-layer">
+                        <div className="flex align-center">
+                            <UserAvatar user={user} style={{backgroundColor: 'transparent'}}/>
+                            <Icon name="chevrondown" className="Dropdown-chevron ml1" size={8} />
                         </div>
-                    </a>
+                    </div>
+                </a>
 
-                    { dropdownOpen ?
+                { dropdownOpen ?
```

Comment:
This is more of a question than a suggestion but would it make sense to use the ` someBool && <youJSXHere>` pattern in this instance rather than doing the ternary with a null? I know we do both but I'm curious if we prefer one over the other and if this might be an instance to start solidifying.